### ISSUE TITLE
[FR] highlight variant stock in parts views

### ISSUE THREAD

**Issue by T0jan** [inventree/InvenTree#5959]
### Please verify that this feature request has NOT been suggested before.

- [X] I checked and didn't find a similar feature request

### Problem statement

hi,

including the variant stock in the stock of the template part is in general a good idea but it causes a bit of confusion for some users on our side. If the part view is used, it is not clear if the displayed stock includes variants until the part is selected and the stock view is checked.

### Suggested solution

Adding an info symbol next to the stock like it is already done in the stock allocation view for build orders probably would clear things up already. It would be premium, if the actual variant stock would be displayed in the tooltip.

### Describe alternatives you've considered

Obviously this is not really necessary as stock information could be directly viewed by going via the stock views and not the part views. However users will always have their weird ways of accessing information and one probably has to tailor into this...

### Do you want to develop this?

- [ ] I want to develop this.

**Comment by SchrodingersGat** [comment]
@T0jan good point, this would be a great thing to implement for the [user interface refactor](https://github.com/inventree/InvenTree/issues/5212)

We could also better highlight the following "quantities":

- Allocated stock
- "External" stock
- Building stock
- Damaged / etc

**Comment by T0jan** [comment]
not stale

**Comment by T0jan** [comment]
not stale

**Comment by RaphaelAdewale** [comment]
Hi @T0jan, can I work on this?

I am a student in the CodePath AI301 program and I want to contribute to this project. I'll start by reproducing the problem locally this week. Any pointers on where to start in the codebase would be appreciated!

**Comment by br413** [comment]
I'd like to work on this as my first contribution.

**Comment by matmair** [comment]
Anyone is free to work on this; just make sure to add a reference to your PR so we can associate the two
